main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -88,10 +88,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.beust.jcommander.Strings;
```

Comment:
@jackyq2015 Why JCommander and not just use [Apache StringUtils methods](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNoneBlank-java.lang.CharSequence...-) for null conversion where you need it ?

Reply:
we do not need this. removed.